yorkie/src/test/kotlin/dev/yorkie/document/crdt/RgaTreeListTest.kt

```diff
@@ -0,0 +1,151 @@
+package dev.yorkie.document.crdt
+
+import dev.yorkie.document.time.ActorID
+import dev.yorkie.document.time.TimeTicket
+import junit.framework.Assert.assertEquals
+import junit.framework.Assert.assertNotSame
+import org.junit.Before
+import org.junit.Test
+
+class RgaTreeListTest {
```

Comment:
It might not cover all the possible cases or violate Kotlin test code convention,
so please let me know if there's something more to add or to fix.